### DISCUSSION TITLE
Buttons positioning in ButtonLayout

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#680] [Proposals]
Actualmente el posicionamiento del ButtonLayout es de esta forma:
![image](https://user-images.githubusercontent.com/6722153/183633109-a4f6b945-73e0-439f-81ba-1155a0fb59fd.png)

Deberíamos posicionar el primary a la izquierda, de manera contrario a lo actual?

@aweell plantea que según dónde se alinee el buttonLayout puede invertir su orden. 
Si las acciones de un dialog van a la derecha, el orden sería:
Link < Secondary < Primary

Si las acciones se posicionan en la izquierda
Primary > Secondary > Link

**Comment by aweell** [comment]
Añado aquí la posibilidad de que se centren, que mantendría los botones igual que a la izquierda.

**Comment by aweell** [comment]
Also discussed in https://github.com/Telefonica/mistica-design/discussions/1590
